packages/server/lib/cloud/api/api.ts

```diff
@@ -9,19 +9,19 @@ const RequestErrors = require('@cypress/request-promise/errors')
 const { agent } = require('@packages/network')
```

Comment:
Not a big deal, but maybe rename this to `index.ts` so that we don't have to do `api/api`?

Reply:
Updated in https://github.com/cypress-io/cypress/pull/29174/commits/58c8daa685a338effff5aa59c98324f65f6242ea